app/models/tx-store/tx-store.ts

```diff
@@ -10,16 +10,19 @@ import BigNumber from "bignumber.js"
 import { withEnvironment } from "../extensions"
 import {
   CosmosMessage,
-  CosmosSendResult,
+  CosmosSigner,
   CosmosTxQueryResult,
 } from "../../services/cosmos"
 import { parseCosmosCoin } from "../../services/cosmos/cosmos.utils"
-import { logError, logMessage } from "../../utils/error"
+import { logError } from "../../utils/error"
 
 import { translateWithFallbackText } from "../../i18n"
 
 import { TxError, TxInsufficientGasFeeError } from "./tx-error"
 
+import { OfflineDirectSigner } from "@cosmjs/proto-signing";
+import { BroadcastTxResponse } from "@cosmjs/stargate"
```

Comment:
Please move this import above relative imports

Reply:
Done